tests/acceptance/features/webUISharingInternalGroupsToRoot/shareWithGroups.feature

```diff
@@ -68,7 +72,8 @@ Feature: Sharing files and folders with internal groups
 
 
   Scenario: share a file with an internal group a member overwrites and unshares the file
-    Given user "Carol" has logged in using the webUI
+    Given user "Carol" has created file "lorem.txt"
+    And user "Carol" has logged in using the webUI
     When the user renames file "lorem.txt" to "new-lorem.txt" using the webUI
```

Comment:
no need to rename here, the test is not about renaming, lets upload directly `new-lorem.txt`